From 97eb11085f7103aa25c6cc2d0306c72d25c87589 Mon Sep 17 00:00:00 2001
From: "chengzhi.hou" <hcz017@gmail.com>
Date: Mon, 4 Jun 2018 10:33:34 +0800
Subject: [PATCH 12/20] SystemUI: Fix systemui crash when showing data usage
 detail

Enable "Set data limit" in setting, after data usage
over data limit, click CellularTile to show data usage
detail will cause systemui crash.

Change-Id: Ib9605f76ca7f40d439cf3e5544974143bcb9b107
---
 .../src/com/android/systemui/qs/tiles/DataUsageDetailView.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/qs/tiles/DataUsageDetailView.java b/packages/SystemUI/src/com/android/systemui/qs/tiles/DataUsageDetailView.java
index 7224ae6a1e1..0961a16502a 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/tiles/DataUsageDetailView.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/tiles/DataUsageDetailView.java
@@ -91,7 +91,7 @@ public class DataUsageDetailView extends LinearLayout {
                     formatBytes(info.usageLevel));
             bottom = res.getString(R.string.quick_settings_cellular_detail_data_limit,
                     formatBytes(info.limitLevel));
-            usageColor = Utils.getDefaultColor(mContext, android.R.attr.colorError);
+            usageColor = Utils.getColorError(mContext);
         }
 
         if (usageColor == 0) {
-- 
2.17.1

